go-update/src/main/java/org/reactome/release/goupdate/GoTermInstanceModifier.java

```diff
@@ -292,18 +308,27 @@ public void deleteGoInstance(Map<String, Map<String,Object>> goTerms, Map<String
 		try
 		{
 			String goId = (String) this.goInstance.getAttributeValue(ReactomeJavaConstants.accession);
+			// A GO term can be deleted if it has a replacement value
 			if (goTerms.get(goId).get(GoUpdateConstants.REPLACED_BY)!= null)
 			{
-				String replacementGOTermAccession = (String) goTerms.get(goId).get(GoUpdateConstants.REPLACED_BY);
+				// If there are multiple replacement options, just use the first one, no clear way to choose a replacement.
+				@SuppressWarnings("unchecked")
+				String replacementGOTermAccession = ((List<String>) goTerms.get(goId).get(GoUpdateConstants.REPLACED_BY)).get(0);
 				// this term has a replacement so we will update all referrers of *this* to point to the replacement.
 				if (allGoInstances.get(replacementGOTermAccession) != null && allGoInstances.get(replacementGOTermAccession).size() > 0)
 				{
 					GKInstance replacementGOTerm = allGoInstances.get(replacementGOTermAccession).get(0);
-					pointAllReferrersToOtherInstance(replacementGOTerm);
+					this.pointAllReferrersToOtherInstance(replacementGOTerm);
 				}
 				deletionStringBuilder.append("Deleting GO instance: \"").append(this.goInstance.toString()).append("\" (GO:").append(goId).append(")\n");
 				adaptor.deleteInstance(this.goInstance);
 			}
+			// A GO term that has no replacement value can still be deleted if it has no referrers.
+			else if (goTerms.get(goId).get(GoUpdateConstants.REPLACED_BY) == null && GoTermsUpdater.getReferrerCounts(this.goInstance).isEmpty())
```

Comment:
The first condition here isn't needed since it would have to be null to fail the if check and go to this else if.